test/models/clinic_test.rb

```diff
@@ -20,27 +17,16 @@ class ClinicTest < ActiveSupport::TestCase
 
     it 'should be unique on name' do
       clinic_name = @clinic.name
-      dupe_clinic = build :clinic, name: clinic_name, created_by: @user
+      dupe_clinic = build :clinic, name: clinic_name
```

Comment:
Another common pattern we'll see a lot - `created_by`s getting dropped. PaperTrail does these by storing everything on version objects rather than the model itself.